SU2_CFD/src/output_structure.cpp

```diff
@@ -8883,6 +8892,920 @@ void COutput::SetEquivalentArea(CSolver *solver_container, CGeometry *geometry,
   
 }
 
+void COutput::WriteSurface_Analysis(CConfig *config, CGeometry *geometry, CSolver *FlowSolver) {
```

Comment:
The MARKER_INTERNAL is a way to define a surface that does not participate on numerical algorithms (time stepping and dissipation) It took me some time to verify that you have exactly the same solution with or without MARKER_INTERNAL. Then I'm using MARKER_ANALYZE to identify which surfaces I want to analyze in detail (average quantities, distortion, etc) It doesn't need to be an internal surface... in fact, we typically analyze the distortion at the fan face which is an inflow BC. Something that you have noticed is that there are several problems in which you need to project the unstructured grid into a structured grid (distortion analysis, interpolation, interaction with other softwares, etc) I'm implementing that kind of projection in a rudimentary way that we should implement and unify in the future.

After several test, we determined that the double surface is the best method to implement an actuator disk (source terms doesn't work OK with unstructured grids... and it has some problems even for structured solvers). SU2 splits the surface internally, so, from the point of view of the user, the input is an internal surface that SU2 transforms into a double side boundary... it is not a very complicated algorithm.